tests/sqlparser_postgres.rs

```diff
@@ -4441,3 +4441,218 @@ fn test_table_unnest_with_ordinality() {
         _ => panic!("Expecting TableFactor::UNNEST with ordinality"),
     }
 }
+
+#[test]
+fn parse_create_trigger() {
```

Comment:
Can we include coverage for following?
- multiple events? (e.g. `INSERT OR UPDATE ...`)
- `EXECUTE PROCEDURE`
- execute with one or more function arguments
- (if the parser supports it), timing like `DEFERRABLE`, `INITIALLY IMMEDIATE` etc'
- referencing without alias (`REFERENCING NEW TABLE FOR ...`)

Would be nice if we could also include negative tests, like invalid combinations or incomplete statements that we expect the parser to reject

Reply:
Hi, I am not too familiar with these cases, could you kindly provide a full sql example of each one so I can make sure those are covered?

Reply:
Now handled multiple events and added several cases in test suite.

Reply:
Also procedure added now.

Reply:
And added testing for multiple function arguments.

Reply:
And added also DEFERRABLE / INITIALLY IMMEDIATE.

Reply:
Added test suite for referencing

Reply:
Added some negative testing